samples/charts/data-chart/stacked-bar-chart/App.razor

```diff
@@ -0,0 +1 @@
+<Infragistics.Samples.Pages.StackedBarChart/>
```

Comment:
replace this line with content of StackedBarChart.razor file

Reply:
done